BaseApp/COD50003.txt

```diff
@@ -0,0 +1,91 @@
+OBJECT Codeunit 50003 DotNet_XslTransform
+{
+  OBJECT-PROPERTIES
+  {
+    Date=;
+    Time=;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      DotNetXslTransform@17024400 : DotNet "'System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Xml.Xsl.XslCompiledTransform";
+
+    [External]
+    PROCEDURE CreateFromBlob@17024400(VAR TempBlob@17024400 : Record 99008535);
+    VAR
+      TransformStream@17024401 : InStream;
+    BEGIN
+      TempBlob.Blob.CREATEINSTREAM(TransformStream);
+      CreateFromStream(TransformStream);
+    END;
+
+    [External]
+    PROCEDURE CreateFromStream@17024424(XslTransformationStream@17024400 : InStream);
+    VAR
+      XmlReader@17024401 : Codeunit 50004;
+    BEGIN
+      XmlReader.CreateFromStream(XslTransformationStream);
+      CreateFromXmlReader(XmlReader);
+    END;
+
+    [External]
+    PROCEDURE CreateFromXmlReader@17024401(VAR XmlReader@17024400 : Codeunit 50004);
+    VAR

```

Comment:
And a 3rd candidate for a helper!

Reply:
Actually, it would be better to use the .Load(String) method and remove the dependency on the XML Reader, as we shouldn't expose the XML Reader functions (see previous comment in XML Reader).